packages/styled-react/src/index.ts

```diff
@@ -0,0 +1,18 @@
+export {
```

Comment:
when does the wrapping happen? 🤔 or as we remove the sx from @primer/react we'd come here and do that wrapping? 👀